contributed_definitions/yaml/NXmpes_peem.yaml

```diff
@@ -0,0 +1,114 @@
+category: application
+doc: |
+  This is a general application definition for a photo-electron emission or low energy electron microscopy/diffraction experiment.
+type: group
+(NXmpes)NXmpes_peem:
+  (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes_peem]
+    (NXmonitor):
+      exists: recommended
+    (NXinstrument):
+      (NXinsertion_device):
+          exists: optional # only when PEEM uses synchrotron light
+      (NXmonochromator): # incoming_beam_monochromator
+          exists: recommended
+      (NXbeam)beam_probe:
+        exists: optional # as we record it in NXsample
+        \@flux: # that will point to the NXmonitor data field
+        incident_energy(NX_FLOAT): # link to monochromator energy
+        incident_polarization_stokes(NX_NUMBER):
+        depends_on:
+        (NXtransformations):
+          direction(NX_number):
+            doc:
+              default is [0,0,1]
+          reference_plane(NX_NUMBER): #how is that for our AC-PEEM?
+              doc:
+                defined as normal to the plane used to measure azimuth. Default is [0,1,0] through the origin
+      (NXoptical_system_em): 
+        field_of_view(NX_FLOAT):
+          doc: |
+            One or two values for size of image on sample or in another space along slow and fast pixel dimensions.
+          \@units:
+        working_distance(NX_FLOAT):
+        magnification(NX_FLOAT):
+      (NXelectronanalyser):
+        (NXcollectioncolumn): # supplement base application definition
+          scheme:
+            doc: |
+              Scheme of the electron collection column.
+            enumeration: [standard, angular dispersive, selective area, deflector, PEEM, PEEM dark-field, LEEM, LEEM dark-field, LEED, dispersive plane, momentum microscope]
```

Comment:
We should add these additional fields also to `NXmpes`. We could also think about making LEEM and LEED a separate appdef, as we would require an incident beam for PEEM and an electron gun for LEEM/LEED. IMO separating this and requiring the appropriate fields for the matching appdef is the more robust and cleaner choice.